src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1238,13 +1238,22 @@ export class DragRef<T = any> {
       : point;
 
     if (this.lockAxis === 'x' || dropContainerLock === 'x') {
```

Comment:
Do we need a test for this code path as well (`lockAxis` + `constrainPosition`)?

Reply:
Yes, was going to do that as well!

Reply:
These have been added as well.